assign_rights/views.py

```diff
@@ -96,6 +96,10 @@ def get_form_cls(self, rights_basis):
             form_cls = StatuteForm
         elif rights_basis == "License":
             form_cls = LicenseForm
+        elif rights_basis == "Policy":
+            form_cls = OtherForm
+        elif rights_basis == "Donor":
+            form_cls = OtherForm
```

Comment:
I think we can just let these additional cases get caught in the `else`, right?

Reply:
You're right 🤦‍♀️